core/src/CrossOrigin.ts

```diff
@@ -687,7 +689,7 @@ class GetElementTransaction extends CrossOriginTransaction<CrossOriginElementDat
                     let isForwardResolved = false;
                     let isResolved = false;
 
-                    (tabster as unknown as Types.TabsterInternal).observedElement!!!.waitElement(name, timeout).then(value => {
+                    (tabster as unknown as Types.TabsterInternal).observedElement!!!.waitElement(name, timeout, accessible).then(value => {
```

Comment:
triple exclamation mark shouldn't be necessary

```suggestion
                    (tabster as unknown as Types.TabsterInternal).observedElement!.waitElement(name, timeout, accessible).then(value => {
```

Reply:
I know. It's a way we used to tell something like «this is a very fragile operator, do not use it at all, but if you use it, make completely sure it's legit».